pkg/security/probe/probe.go

```diff
@@ -169,12 +210,21 @@ func (p *Probe) HandleActions(rule *rules.Rule, event eval.Event) {
 }
 
 // AddEventConsumer sets a probe event consumer
-func (p *Probe) AddEventConsumer(eventType model.EventType, handler EventConsumer) error {
-	if eventType >= model.MaxAllEventType {
-		return errors.New("unsupported event type")
+func (p *Probe) AddEventConsumer(consumer EventConsumer) error {
```

Comment:
don't we need to lock before adding/removing/accessing consumers?

Reply:
Consumer are added at the very beginning in the call flow and there is no way to modify the list after.